oeps/processes/oep-0062-proc-working-groups.rst

```diff
@@ -0,0 +1,218 @@
+* Generate and review work artifacts (code, documents, images, decisions, etc.) in publicly-visible spaces.
+* Publicize any and all meeting times in UTC, with an open invite to Open edX community members.
+* Maintain an up-to-date home page under the `Working Groups space`_ on the Open edX wiki, including:
+
+  * the group's name and purpose,
+  * the current chairperson and other assigned roles,
+  * any meeting times and video conferencing links,
+  * links to any communication channels or task trackers,
+  * links to any spaces containing work artifacts, and
+  * any additional documentation on group structure and operations.
+
+Working groups may exempt themselves from certain guidelines when they would compromise their ability to do work related to:
+
+* undisclosed security issues, or
+* unreleased marketing materials.
+
+Such groups should otherwise follow the openness & inclusivity guidelines to their best effort.
+
+Finally, as with any activity in the Open edX community, working groups are subject to the `Open edX Code of Conduct`_.
+
+
+Structure and operation
+-----------------------
+
+Beyond the guidelines above, working groups have complete autonomy in their internal structure and operating model. For example:
+
+* they may define specific membership criteria, or deem membership to be fluid;
+* they may define and assign additional roles, or choose to only have a chairperson;
+* they may break themselves into teams or operate as a single unit;
+* and so on.
+
+What do working groups do?
+==========================
+
+Generally speaking, working groups can contribute to the Open edX project in any way. For example, they might:
+
+* coordinate amongst its various members' projects;
+* help design and implement platform features;
+* create workshops or courses;
+* write documentation and architectural decision records (ADRs);
+* triage issues;
+* assist community members; or
+* simply discuss an aspect Open edX platform.
+
+Certain working groups may step up to fulfill a responsibility called for in an Open edX Proposal. For example, at the time of writing, the Product Working Group is leading the implementation `the Core Product from OEP-57`_, the DevOps Working Group handles `the release process from OEP-10`_, and the Architecture Coordination Working Group facilitates `the architecture advisory process from OEP-56`_. Due to the fluid nature of working groups, the association between OEP-declared responsibilities and specific working group names is subject to change over time.
+
+Furthermore, working groups can step up to support maintenance of Open edX components, per the process detailed in `OEP-55 <https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055-proc-project-maintainers.html>`_.
+
+How are working groups formed and dissolved?
+============================================
+
+Forming
+-------
+
+To form a working group, any community member can share an announcement describing:
```

Comment:
Do you want to specify where to announce? I think ideally that would be able to reference another doc about "how to announce things to the whole community", which may already exist?

Reply:
A confluence doc is a good idea. That way, if we change how we announce things, we won't need to remember to come back and update this section.